<?xml version="1.0"?>
<launch>
  <arg name="use_rear_wheel_speed" default="true"/>
  <arg name="use_adaptive_gear_ratio" default="true"/>
  <arg name="command_timeout" default="1000"/>
  <arg name="loop_rate" default="30.0"/>
  <arg name="acceleration_limit" default="3.0"/>
  <arg name="deceleration_limit" default="3.0"/>
  <arg name="max_curvature_rate" default="0.75"/>
  <arg name="steering_offset" default="0.0"/>

  <node pkg="as" type="ssc_interface" name="ssc_interface" output="screen">
    <param name="use_rear_wheel_speed" value="$(arg use_rear_wheel_speed)" />
    <param name="use_adaptive_gear_ratio" value="$(arg use_adaptive_gear_ratio)" />
    <param name="command_timeout" value="$(arg command_timeout)" />
    <param name="loop_rate" value="$(arg loop_rate)" />
    <param name="acceleration_limit" value="$(arg acceleration_limit)" />
    <param name="deceleration_limit" value="$(arg deceleration_limit)" />
    <param name="max_curvature_rate" value="$(arg max_curvature_rate)" />
    <param name="steering_offset" value="$(arg steering_offset)" />
  </node>
</launch>
